docs/release-notes/2.33.0.md

```diff
@@ -0,0 +1,145 @@
+
+## Release Highlights
+
+
+### High Availability in PMM
+
+
+We are excited to share our roadmap for High Availability (HA) in PMM, a new solution that our users have been anticipating. HA will provide your PMM near-zero downtime and wil ensure monitoring without data loss during a failover.
+
+
+To ensure a smooth rollout, HA in PMM will be introduced in phases, in line with your requirements and feedback, with PMM 2.33.0 being the starting point. 
+
+!!! caution alert alert-warning "Important"
+    PMM HA will be rolled out in three stages. Stage one is included with PMM 2.33.0 and stages two and three will be rolled out next year.
+
+The following features/use cases will be covered as part of the various stages of HA in PMM:
+
+
+|**Stages of HA in PMM**              | **Solutions Provided**          |
+|-------------------------------------|---------------------------------|
+|**Stage one**           |As an initial step towards preventing data loss we have developed the following:<br><br>**Prevent data loss with VM integration for short outages**<br>If PMM Server goes down, metrics will be buffered on the monitored instance until PMM Server comes back online, which is when metrics will get pushed to PMM Server. Note that this only applies to PMM Clients which run in the so-called `push` mode. The same holds true when a brief network outage occurs on the client side.|
+|**Stage two**           |As part of PMM HA stage two in HA we plan to implement the following:<br><br>**HA data sources**<br>As part of stage two we will let the users use external data sources therby decreasing dependency on the file system.
+|**Stage three**         |As part of PMM HA stage three we plan to implement the following:<br><br>**Clusterized HA**<br>Clustered PMM will be the focus of stage three. Detailed information will be included in the upcoming release notes.
+
+
+
+
+
+
+### Guided tour of Percona Alerting
+
+
+Following the recent improvements to PMM Alerting, we have added a short tutorial to help you explore our streamlined alerting system based on Grafana.
+The in-app tutorial automatically pops up when you first open the **Alerting** page to help you uncover the fundamentals of Percona Alerting.
+
+
+ 
+### Restore MongoDB backups easier
+
+Building on the significant improvements for MongoDB Backup Management introduced in the previous release. We are now simplifying the process for restoring physical MongoDB backups.
+Starting with this release, you can restore physical backups straight from the UI and PMM will handle the process end-to-end. Prior to this, you would require to perform additional manual steps to restart your MongoDB database service so that your applications could make use of the restored data.
+
+### Components Upgrade
```

Comment:
I do not know whether we should put upgrades to the highlight section. @rnovikovP what do you think?